src/var/isFunction.js

```diff
@@ -7,7 +9,12 @@ define( function() {
 		// In some browsers, typeof returns "function" for HTML <object> elements
 		// (i.e., `typeof document.createElement( "object" ) === "function"`).
 		// We don't want to classify *any* DOM node as a function.
-		return typeof obj === "function" && typeof obj.nodeType !== "number";
+		// Support: QtWeb <=3.8.5, WebKit <=534.34, wkhtmltopdf tool <=0.12.5
+		// Plus for old WebKit, typeof returns "function" for HTML collections
+		// (e.g., `typeof document.getElementsByTagName("div") === "function"`). (gh-4756)
+		return typeof obj === "function" && typeof obj.nodeType !== "number" &&
+			[ "[object HTMLCollection]", "[object NodeList]" ]
+				.indexOf( toString.call( obj ) ) === -1;
```

Comment:
I'd like to see something a bit less fine-tuned.
```suggestion
			typeof obj.item !== "function";
```